Cargo.toml

```diff
@@ -43,6 +43,7 @@ image = { version = "0.24", optional = true, default-features = false, features
 log = "0.4"
 x11rb = { version = "0.12" }
 wl-clipboard-rs = { version = "0.8", optional = true }
+os_pipe = { version = "1.1.5" }
```

Comment:
We don't really need anything from this crate, but one of the structs from this crate gets returned by a method in `wl-clipboard-rs` and we need to include it in one of our function signatures.